### PR TITLE
fix: map GEF V3 status to V2 UI labels for Portal

### PR DESCRIPTION
**Pull request by pbradley-f4**

### CODE REVIEW

File: portal/templates/_macros/status-label.njk [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    UKEF_IN_PROGRESS: "In progress by UKEF",
+    UKEF_ACCEPTED_CONDITIONAL: "Accepted by UKEF (with conditions)",
+    UKEF_ACCEPTED_UNCONDITIONAL: "Accepted by UKEF (without conditions)",
+    UKEF_DECLINED: "Rejected by UKEF",
+    EXPIRED: "Expired",
+    WITHDRAWN: "Withdrawn"
+  }
+%}
+
+{% macro getLabel(status) %}
```
Comment (davehlewis) [inline portal/templates/_macros/status-label.njk:17]: Would this not be better as a filter than a macro? much easier to use via pipe function I think

Comment (pbradley-f4) [reply portal/templates/_macros/status-label.njk:17]: I can change it if you feel strongly enough ... just say the word

Comment (davehlewis) [reply portal/templates/_macros/status-label.njk:17]: Given the reuse elsewhere definitely feels the best way - sorry to be annoying about it

Comment (pbradley-f4) [reply portal/templates/_macros/status-label.njk:17]: its cool .... will make it so

**Review by davehlewis — APPROVED** [review]
LGTM

**Pull request merged**